lib/helper.rb

```diff
@@ -0,0 +1,27 @@
+module Helperable
+  def valid_move?(board, index)
+    true if index.between?(0, 8) && !position_taken?(board, index)
+  end
+
+  def position_taken?(board, indx)
+    if board[indx] == 'X' || board[indx] == 'O'
+      true
+    else
+      false
+    end
+  end
+
+  def player_names()
+    gets.strip
+  end
+
+  def next_line()
+    puts "\n \n"
```

Comment:
- [ ] No `puts/print` and `get.chomp` outside `main.rb` file. 

line #19 you could just return the string and use **`puts next_line`** in the main instead